gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/FsBenchmark.java

```diff
@@ -241,10 +246,14 @@ private void benchmarkWrite(
                         output.write(writeBuffer);
                         fileBytesWrite += writeSize;
                         writeCallBytes.accept(writeSize);
+
+                        writeFileTime.add(System.nanoTime() - writeCallStart);
                         writeCallTimeNs.accept(System.nanoTime() - writeCallStart);
                       } while (fileBytesWrite < totalSize);
 
                       writeFileBytes.accept(fileBytesWrite);
+
+                      writeFileTime.add(System.nanoTime() - writeStart);
```

Comment:
Wont this add twice?

Reply:
Can you elaborate on what can be added twice ?

Reply:
modified